Widok skanowania

maciek1651g opened the pull request:
Gotowy widok skanowania #64 
![widok_skanowania](https://user-images.githubusercontent.com/45820718/78978847-eafc7080-7b1a-11ea-9be8-30194e8d5501.png)
![widok_skanowania_anuluj](https://user-images.githubusercontent.com/45820718/78978870-f64f9c00-7b1a-11ea-9fd2-b8f1d4ec2f7c.png)

maciek1651g committed:
Utworzenie struktury widoku skanowania

maciek1651g committed:
Zmiana nazwy widoku

maciek1651g committed:
Dopracowanie wyglądu widoku

maciek1651g committed:
Merge branch 'develop' into widok_skanowania

maciek1651g committed:
Poprawki w wyglądzie przycisków

Wprowadzono wyskakujące okienko do przycisku anuluj

maciek1651g committed:
Merge branch 'develop' into widok_skanowania

maciek1651g committed:
Zaimplementowanie widoku z kamery i skanowania kodów

maciek1651g commented:
zapomniałem w pliku App.xaml.cs zmienić stronę startową na LoginPage

maciek1651g committed:
Naprawa skanera kodów

naprawa poprzez obniżenie wersji bibliotek zXing do wersji 2.3.1

maciek1651g committed:
Dodanie ikonek do przycisków

MikolajczukKamil commented:
Ogólnie jest dobrze, ale są pewne aspekty do dopracowania
- w katalogu projektu będzie katalog views i w nim poszczególne widoki, pozwoli to utrzymać przejrzystą strukturę, trzeba przenieść widok do views/view-scanning/
- należałoby zwrócić uwagę na formatowanie kodu i poprawić to tak aby było to czytelne, tu przykłady

![fajnie](https://user-images.githubusercontent.com/25321219/79045854-b1586200-7c0d-11ea-8dec-f927794d9c6e.PNG)

![lub to](https://user-images.githubusercontent.com/25321219/79045899-f5e3fd80-7c0d-11ea-85d4-be035c8ef2ee.PNG)

chyba widać nie wygląda to fajnie 

- co do samego skanowania, działać działa ale dobrze by było aby blokować kolejne akcje jeżeli kod się nie zmienił
![wynik](https://user-images.githubusercontent.com/25321219/79045857-b2898f00-7c0d-11ea-8a01-4e8245f1da89.jpg)

W ramach ciekawostek czasami ten skaner wyrzuca chyba losowe liczby mam nadzieję że nie zepsuje to zabawy 
![a to ciekawe](https://user-images.githubusercontent.com/25321219/79046072-d699a000-7c0e-11ea-9c42-1be6519db51b.jpg)

MikolajczukKamil commented:
Przydał by się widok listy już zeskanowanych elementów i możliwością usunięcia ewentualnych błędnie zeskanowanych elmentów, może warto zmienić napis zakończ na podgląd co by wyświetlało widok raportu ze spakowania z możliwością powrotu do dalszego skanowania

maciek1651g committed:
Wprowadzenie poprawek

Dodano folder views. Zapamiętuję ostatni zeskanowany element w celu zatrzymania dalszego skanowania tego samego. TryHarder=false, by nie wyrzucało losowych liczb. Lekka poprawa formatowania kodu.

maciek1651g commented:
Co do formatowania kodu w pliku xmlns to nie mogę wyrzucić lejbeli i tego towarzystwa poza znacznik Grid.

Jerry-BloodBerry reviewed (approved):
Mi się podoba :D Odkryłem, że jak przekrzywiam telefon o ok. -30 stopni to skanuje lepiej, ale to xiaomi więc ma odpały. Dla mnie już jest dobrze. Niech jeszcze Kamil się wypowie

MikolajczukKamil commented:
> Co do formatowania kodu w pliku xmlns to nie mogę wyrzucić lejbeli i tego towarzystwa poza znacznik Grid.

@maciek1651g 
Oczywiście, chodziło mi o formatowanie czyli wcięcia aby nie robić tasiemca w jednej linii tylko jak jest dużo właściwości to dać je w kolejnych liniach tak jak tam na górze (często podaje się żeby linia miała do 80 znaków nie trzeba mierzyć linijką oczywiście tylko tak aby mieściło się na 1 ekranie)

O tak jak tu zrobiłem 1 label
![o tak](https://user-images.githubusercontent.com/25321219/79051663-a9121e00-7c31-11ea-9efa-bafb75e3a595.PNG)

maciek1651g committed:
Wprowadzono nowy alert skanowania oraz poprawiono formatowanie kodu

maciek1651g committed:
Usunięcie niepotrzebnych komentarzy i wywołań funkcji

maciek1651g commented:
Nie wiedziałem jak wprowadzać dalsze zmiany i postanowiłem dokończyć ten wygląd, a podgląd zeskanowanych elementów zrobić w innej gałęzi. Więc czekam na opinię @MikolajczukKamil jak to powinno wyglądać. Druga gałąź #79

MikolajczukKamil reviewed (approved):
Jest miodzio tylko się konflikt zrobił

maciek1651g committed:
Merge branch 'develop' into widok_skanowania
